src/registry_schemas/example_data/__init__.py

```diff
@@ -90,6 +91,7 @@
     'FILING_HEADER',
     'FILING_TEMPLATE',
     'FILINGS_WITH_TYPES',
+    'FIRMS_CONVERSION',
```

Comment:
Would it better to name this "CONVERSION" to be in sync with the schema name (same as the others)?

Reply:
Oh, I see there's already a CONVERSION entry here... Is that for Coops then?

Reply:
That one is for Benefit company. I renamed it to BEN_CONVERSION to be clear. I am not sure whether other entity types in corps world have share structure. The schema is generic with optional elements and we are planning to add entity type specific validation in the validator.